src/dev/jest/config.js

```diff
@@ -44,6 +44,7 @@ export default {
     'packages/kbn-ui-framework/src/services/**/*.js',
     '!packages/kbn-ui-framework/src/services/index.js',
     '!packages/kbn-ui-framework/src/services/**/*/index.js',
+    'src/legacy/core_plugins/metrics/**/*.js'
```

Comment:
Interesting.., but not sure that we should fix it only for TSVB.
I prefer to see something  like `'/src/**/*.{js,ts,tsx}'` but not sure that the coverage report will be correctly generated for ts files...

@spalger do you know why we don't include files from the 'roots' field to our coverage report?

Reply:
Mostly because the coverage information provided by Jest is only interesting to a tiny amount of people and the console presentation is very sub-optimal, but I don't see any problem turning it on if someone would find value in it.

I am personally unconvinced that most coverage reports are valuable, but I don't want to keep people from doing things they find value in.

Reply:
If we apply the `'/src/**/*.{js,ts,tsx}'` pattern the process of making coverage for a single module will take a lot of time. So it would be comfortable to leave it as is and who are interested in would be able to add its patterns too.

Reply:
ok... let's turn on it only for TSVB.